.eslintrc.yml

```diff
@@ -76,3 +76,10 @@ overrides:
       strict: off
     globals:
       run_spec: false
+  - files:
+      - src/cli/**/*.js
+    rules:
+      no-restricted-modules:
```

Comment:
I've noticed that you added bunch of disables for this rule in the code, so the question is how important it is to have it here?! I would suggest to remove this rule.

Reply:
The idea is to disallow importing anything in `cli` from other directories except for two modules: `../index` and `../common/third-party`. But even though the rule supports negative patterns for specifying such exceptions (e.g. `"!baz-module/good"`), internally it uses the [`ignore`](https://www.npmjs.com/package/ignore) module, which doesn't play well with relative paths that contain `..`. The reason is that that module is an implementation of the `.gitignore` rules where such paths can't be used. So the negative pattern `!../common/third-party` doesn't work. `!../index` seems to work though.

BTW, Prettier uses the same `ignore` module and exhibits similar bugs if the file specified by the `--ignore-path` CLI option isn't in the root of the project. We need to open an issue about that.